examples/patch_on_update.rs

```diff
@@ -1,14 +1,26 @@
 use bevy::prelude::*;
 use bevy_simple_subsecond_system::prelude::*;
+use bevy_simple_subsecond_system::{AppWrapper, ReloadableAppExt};
+
 fn main() {
     App::new()
         .add_plugins(DefaultPlugins)
         .add_plugins(SimpleSubsecondPlugin::default())
         .add_systems(Startup, setup)
         .add_systems(Update, configure_ui)
+        .reloadable(|mut app: AppWrapper| -> AppWrapper {
+            app.add_systems(Update, do_thing);
```

Comment:
I wonder how this would behave if we add a fixed update system in here. Judging from the implementation, it should work, right?